### ISSUE TITLE
The 'cashTollOnly' RoadClasses value causes directions request to fail.

### ISSUE THREAD

**Issue by kpennacchia** [mapbox/mapbox-directions-swift#779]
Attempting to request directions while adding `cashTollOnly` to the exclusions causes the request to fail with the following error.

> Exclude value must be one of: toll, motorway, ferry, unpaved, cash_only_tolls, border_crossing, point(lon lat)

Example code snippet:

```swift
let routeOptions = NavigationRouteOptions(waypoints: waypoints, profileIdentifier: .automobile)
routeOptions.roadClassesToAvoid = [.cashTollOnly]
```

It looks like the string value being used is `cash_only_toll`, but the correct value is `cash_only_tolls`.

**Comment by mickbyrne** [comment]
Same here, needs to be fixed!

**Comment by Udumft** [comment]
Hi! The fix has landed in [this PR](https://github.com/mapbox/mapbox-directions-swift/pull/801) and should be available in next version release (2.10).
Thank you for flagging the issue!
